test/stylelint/extends/extends.out.css

```diff
@@ -30,6 +30,7 @@
   width: 30%;
 }
 /* Single-line comment */
+
```

Comment:
Why do you add 1 brank line after comments? It seemes not to be related to this PR, right?